appveyor.yml

```diff
@@ -66,24 +66,18 @@ test_script:
 
       if ($testsExitCode -ne 0) {exit $testsExitCode}
 
-  - ps: |
-      & coverage erase
+  - coverage erase --rcfile=.coveragerc_appveyor
+  - del /Q coverage.xml
 
-      $clcachePy = "$((Get-Location).Path)\clcache.py"
-      $env:CLCACHE_CMD = "coverage run -a --branch --include=$clcachePy $clcachePy";
-      & py.test --junitxml .\integrationtests.xml integrationtests.py --cov=clcache
+  - ps: |
+      & py.test --junitxml .\integrationtests.xml integrationtests.py --cov=clcache --cov-config=.coveragerc_appveyor
```

Comment:
> Does the plugin for py.test magically hook into subprocesses

Yes. This is why all my manual confuguratiions did not work and why running without pytest worked perfectly.

The general process is documented here: http://coverage.readthedocs.io/en/coverage-4.2/subprocess.html This is used by pytest-cov as indicated by

> For subprocess measurement environment variables must make it from the main process to the subprocess. The python used by the subprocess must have pytest-cov installed. The subprocess must do normal site initialisation so that the environment variables can be detected and coverage started.

https://github.com/pytest-dev/pytest-cov#limitations

and

> Note that this plugin controls some options and setting the option in the config file will have no effect. These include specifying source to be measured (source option) and all data file handling (data_file and parallel options).

https://github.com/pytest-dev/pytest-cov#coverage-config-file